### PR TITLE
Add find_package support for XRT for embedded platforms

### PR DESCRIPTION
**Pull request by rbramand-xilinx**
> This fix resolves cr - https://jira.xilinx.com/browse/CR-1110775
> to use find_package(XRT) users should install xrt-dev package

### CODE REVIEW

File: src/CMake/config/xrt-edge.fp.in [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  endif()
+endif()
+
+get_filename_component(@PROJECT_NAME@_CMAKE_DIR "${CMAKE_CURRENT_LIST_DIR}" ABSOLUTE)
+
+set(@PROJECT_NAME@_CORE_LIBRARIES @PROJECT_NAME@::xrt_core @PROJECT_NAME@::xrt_coreutil)
+set(@PROJECT_NAME@_COREUTIL_LIBRARIES @PROJECT_NAME@::xrt_coreutil)
+set(@PROJECT_NAME@_OPENCL_LIBRARIES @PROJECT_NAME@::xilinxopencl @PROJECT_NAME@::xrt++)
+set(@PROJECT_NAME@_SWEMU_LIBRARIES @PROJECT_NAME@::xrt_swemu)
+set(@PROJECT_NAME@_HWEMU_LIBRARIES @PROJECT_NAME@::xrt_hwemu)
```
Comment (chvamshi-xilinx) [inline src/CMake/config/xrt-edge.fp.in:45]: Please add xrt_coreutil to SWEMU_lIBRARIES and HWEMU_LIBRARIES also.

**Comment by chvamshi-xilinx** [comment]
Overall it is looking good. Just one minor comment.

**Review by stsoe — APPROVED** [review]
Looks good.  Thanks

File: src/CMake/config/xrt-edge.fp.in (hunk 1 above)
Comment (rbramand-xilinx) [reply src/CMake/config/xrt-edge.fp.in]: @chvamshi-xilinx ,made changes as requested please review

**Review by chvamshi-xilinx — APPROVED** [review]

**Comment by bryanloz-xilinx** [comment]
Thanks Everyone! Much Love from Vitis-AI!

**Pull request merged**